src/components/projects/create-wp-form/create-wp-form/create-wp-form.test.tsx

```diff
@@ -3,24 +3,36 @@
  * See the LICENSE file in the repository root folder for details.
  */
 
-import { render, screen } from "../../../../test-support/test-utils"
-import CreateWPFormView from "./create-wp-form";
+import { render, screen } from '../../../../test-support/test-utils';
+import CreateWPFormView from './create-wp-form';
 
-const mockDependencies = ["1.2.3", "1.2.1", "1.12.0"];
-const mockEA = ["yooooo", "wassup", "bruh", "whoosh"];
-const mockDeliverables = ["go vroom"];
+const mockDependencies = ['1.2.3', '1.2.1', '1.12.0'];
+const mockEA = ['yooooo', 'wassup', 'bruh', 'whoosh'];
+const mockDeliverables = ['go vroom'];
 const mockUtils = {
   add: () => null,
   remove: () => null,
   update: () => null
 };
+const mockStateBundle = {
```

Comment:
Pretty sure these can go in the proper test support utils stub file https://github.com/Northeastern-Electric-Racing/PM-Dashboard-v2/blob/main/src/test-support/test-data/test-utils.stub.ts